src/pages/LoginPage.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="login">
+    <div class="login__title">
+      <SectionTitle variation="40-60-80">Hi there!</SectionTitle>
+      <SectionTitle color="bombay" variation="40-60-80"
+        >Care to log in?</SectionTitle
+      >
+    </div>
+    <FormCard @submit="handleSubmit">
```

Comment:
You can use here `@submit.prevent` and there's no need to call `event.preventDefault()` in `handleSubmit` method

Reply:
You're right. Now when I looked at it I realized, I don't have to do it at all, because it's already prevented in FormCard component: 
`    <form
      class="form-card__form"
      @submit.prevent="$emit('submit', $event)"`